runtime/primitives/src/currency.rs

```diff
@@ -112,11 +113,41 @@ impl CurrencyId {
 		pub const KSM: CurrencyId = CurrencyId(4);
 		pub const PBLO: CurrencyId = CurrencyId(5);
 
-		/// Karura stable coin (Karura Dollar), not native.
+		// Non-Native Tokens (101 - 1000)
+		/// Karura stable coin(Karura Dollar), not native.
 		#[allow(non_upper_case_globals)]
 		pub const kUSD: CurrencyId = CurrencyId(129);
 		pub const USDT: CurrencyId = CurrencyId(130);
 		pub const USDC: CurrencyId = CurrencyId(131);
+		/// Wrapped BTC
+		#[allow(non_upper_case_globals)]
+		pub const wBTC: CurrencyId = CurrencyId(132);
+		/// Wrapped ETH
+		#[allow(non_upper_case_globals)]
+		pub const wETH: CurrencyId = CurrencyId(133);
+		/// Acala Dollar
+		#[allow(non_upper_case_globals)]
+		pub const aUSD: CurrencyId = CurrencyId(134);
+
+		// Staked asset xTokens (1001 - 100_000_000_000)
```

Comment:
What does the `100_000_000_000` in this comment mean?

Reply:
The upper limit to the rage available to this type of token. This can be changed, I chose it because this is at the bottom of the set ranges in CurrencyFactory.

Reply:
hmm but allowing that upper bound would mix the two ranges for x tokens and fnft right?

Reply:
You're right. Good catch. I forgot to update this
0d21e689588a4796cfe0fab8b91f86e450360276